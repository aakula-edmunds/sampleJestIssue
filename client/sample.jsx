import React from 'react';


export function SampleOne(props) {
  return <div className="first"><p>Sample One</p></div>;
}
