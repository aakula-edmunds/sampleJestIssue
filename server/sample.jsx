import React from 'react';


export function SampleServerOne(props) {
  return <div className="first"><p>Sample Server One</p></div>;
}
