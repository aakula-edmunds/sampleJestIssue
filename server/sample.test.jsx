import React from 'react';
import { SampleServerOne } from './sample';


describe('server/sample.jsx', () => {
  test('should work', () => {
    const x = shallow(<SampleServerOne />);
    expect(x).toMatchSnapshot();
  });
});
