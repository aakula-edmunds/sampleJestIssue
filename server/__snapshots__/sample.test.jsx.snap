// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`server/sample.jsx should work 1`] = `
<div
  className="first"
>
  <p>
    Sample Server One
  </p>
</div>
`;
